geode-core/src/main/java/org/apache/geode/cache/query/security/MethodInvocationAuthorizer.java

```diff
@@ -58,4 +59,6 @@
    *         {@code false} otherwise.
    */
   boolean authorize(Method method, Object target);
+
+  default void initialize(Cache cache, Set<String> parameters) {}
```

Comment:
Would you be able to add java doc here to explain that a public no arg constructor needs to be added to the implementing class and what this method is for?

Reply:
Done